packages/react/src/components/fileInput/FileInput.tsx

```diff
@@ -0,0 +1,554 @@
+import path from 'path';
+
+import React, { ChangeEvent, useEffect, useRef, useState } from 'react';
+
+// import core base styles
+import 'hds-core';
+import composeAriaDescribedBy from '../../utils/composeAriaDescribedBy';
+import classNames from '../../utils/classNames';
+import { Button } from '../button';
+import { IconPlus, IconPhoto, IconCross, IconDocument, IconUpload } from '../../icons';
+import { InputWrapper } from '../../internal/input-wrapper/InputWrapper';
+import styles from './FileInput.module.scss';
+
+type FileProperty = keyof File;
+
+const isEqualFileBy = (givenProperties: FileProperty[], a: File, b: File): boolean => {
+  const sameProps: FileProperty[] = givenProperties.filter((property: FileProperty) => a[property] === b[property]);
+  return sameProps.length === givenProperties.length;
+};
+
+const findDuplicateByNameAndType = (files: File[], fileToCompare: File): File | undefined =>
+  files.find((file: File) => isEqualFileBy(['name', 'type'], file, fileToCompare));
+
+type DragAndDropProps = {
+  label: string;
+  helperText: string;
+};
+
+type Language = 'en' | 'fi' | 'sv';
```

Comment:
This seems to be a common pattern in other HDS components too, that we support 3 languages. We discussed before that it does have the bad side-effect of limiting the usage only to services that has these languages. I wonder if we should not limit to 3 languages, but let users give those messages through props? Though there's quite many texts there that the user should then give. When the component gives them based on the language that has been selected, it automates things a lot. I wonder if we should still enable users overwriting those texts some way 🤔

Reply:
I would restrict the languages because this is not the only component with internal texts. 
Another option is to make all ten labels configurable. I tested the API of component, and it starts to be a bit annoying to use 🤔

Reply:
But on the other hand, it could make texts more apparent for the library user 🤔